app/views/handlers/list_add_question.py

```diff
@@ -1,15 +1,34 @@
-from typing import MutableMapping
+from typing import Any, MutableMapping
+
+from flask import url_for
 
 from app.views.handlers.list_action import ListAction
 
 
 class ListAddQuestion(ListAction):
+    def __init__(self, *args: Any):
+        self._list_item_id: str | None = None
+        super().__init__(*args)
+
     def is_location_valid(self):
         if not super().is_location_valid() or self._current_location.list_item_id:
             return False
         return True
 
     def get_next_location_url(self):
+        if self._list_item_id and (
```

Comment:
Just had a thought - when would `self._list_item_id` actually be none?

Reply:
ahhh I see, so you'd end up being routed back to the 'any other items to add' block if it was misused. I imagine we'd have a lot of issues with progress in that case, as blocks wouldn't be marked complete before finding next incomplete url, but not against doing this